sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryServicesImpl.java

```diff
@@ -174,6 +175,8 @@ public class BigQueryServicesImpl implements BigQueryServices {
   // We set it to 9MB, which leaves room for request overhead.
   private static final Integer MAX_BQ_ROW_PAYLOAD = 9 * 1024 * 1024;
 
+  private static final String MAX_BQ_ROW_PAYLOAD_DESC = "9MB";
```

Comment:
this could be derived by MAX_BQ_ROW_PAYLOAD

Reply:
updated, thanks